src/cc2olx/olx.py

```diff
@@ -226,10 +226,23 @@ def _create_olx_nodes(self, element_data: dict) -> List["xml.dom.minidom.Element
             return self._fallback_olx_nodes
 
         for content_processor in self._content_processors:
-            if olx_nodes := content_processor.process(resource, idref):
-                for olx_node in olx_nodes:
-                    self._post_process(olx_node)
-                return olx_nodes
+            try:
+                olx_nodes = content_processor.process(resource, idref)
+            except Exception as exc:
+                logger.error(
+                    'An error occurred during resource "%s" processing by %s:',
+                    idref,
+                    type(content_processor).__name__,
```

Comment:
Comment (no action required): This is fine, but if your processors have any potentially useful per-instance information that they want to expose in their `__repr__`, then logging the instance and allowing it to put its information in the log message would be useful.